src/stylesheets/layout/_first.scss

```diff
@@ -73,9 +90,26 @@ $textFontSize: 22px;
     border-radius: 4px;
     color: v(bg-clr);
     transform: rotate(45deg);
+  }
 
-    &:hover {
-      cursor: pointer;
-    }
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.bg-image {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background: url('../images/first_bg.jpeg') no-repeat center;
```

Comment:
Use img tag instead of background-image. I know, that it's semantically better, but img tag has much more benefits. Also checkout `background-img` mixin  (or smth like this)  in _mixins.scss.